exercises/concept/calculator-conundrum/.meta/src/reference/java/CalculatorConundrum.java

```diff
@@ -0,0 +1,30 @@
+public class CalculatorConundrum {
+    private final String answer;
+
+    public CalculatorConundrum(int operand1, int operand2, String operation) {
+        int result;
+        try {
+            if (operation.equals("+")) {
```

Comment:
I'd use a switch-case here: https://www.geeksforgeeks.org/string-in-switch-case-in-java/

You have to check if operation is null before the switch-case, but that seems like a reasonable cost to get a clearer flow (and allows that `default` case to be more obvious).